### PR TITLE
Update 02-robots.md

### PR DESCRIPTION
**Pull request by okada-blue**
Update "Robots" section of learn.robosats.com

Please feel free to modify text and/or formatting as needed. Apologies in advance for any non-standard contribution practices!

### CODE REVIEW

File: docs/_pages/docs/03-understand/02-robots.md [hunk 1]
```diff
@@ -10,4 +10,48 @@ sidebar:
 src: "_pages/docs/03-understand/02-robots.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+{% include wip %}
```
Comment (Reckless-Satoshi) [inline docs/_pages/docs/03-understand/02-robots.md:13]: We can now get rid of this line, it displays the "Work in progress" notice.

**Review by Reckless-Satoshi — CHANGES_REQUESTED** [review]
Hey @okada-blue,

Thanks a lot for giving a push to the Documentation work. Congratulations on your first PR ever! The text reads really well.

I have left a comment for a small change (just a line deletion). Will also serve as practice for more complex reviews :)

Let's make this the first ever non-dev rewarded PR. Please submit an invoice for 20K Sats. It's little, just a prove of appreciation, but one more step towards to working for Bitcoin :D

File: docs/_pages/docs/03-understand/02-robots.md (hunk 1 above)
Comment (okada-blue) [reply docs/_pages/docs/03-understand/02-robots.md:13]: Ah I didn't realize what that line intended

Yay! That's awesome, thank you very much :D let me try out this invoice:

lnbc200u1p3s5v32pp5p62jxach58d48u6yg4tt7jms9pvs7m07j4m7dzk2stud6pa62u8qdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5r7ta253wcwp8usfjdqf5tja474wx877lu49puvqhn5gktud50hvs9qyyssq3cygts9l8r2rhzrv89a6d4wz8ect8296fsp60y32y4k00ghav5fkqm8dvlgme49mjvvcn0a3s8kuw7mzxs4jkltjtn0g4ufxm28c5xqp2txpxu

**Comment by okada-blue** [comment]
> Please submit an invoice for 20K Sats

Woops, I provided the invoice on the commit comment. I've copied here as well: lnbc200u1p3s5v32pp5p62jxach58d48u6yg4tt7jms9pvs7m07j4m7dzk2stud6pa62u8qdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5r7ta253wcwp8usfjdqf5tja474wx877lu49puvqhn5gktud50hvs9qyyssq3cygts9l8r2rhzrv89a6d4wz8ect8296fsp60y32y4k00ghav5fkqm8dvlgme49mjvvcn0a3s8kuw7mzxs4jkltjtn0g4ufxm28c5xqp2txpxu

File: docs/_pages/docs/03-understand/02-robots.md [hunk 2]
```diff
@@ -10,4 +10,48 @@ sidebar:
...
+
+## **Robot Construction Process**
+
+RoboSats references the source code for RoboHash.org as a quick way to generate fresh avatars for a website. Your robot is "built" from a unique token, a random string of characters (ZD3I7XH...), where only those characters in their exact order can generate that exact robot avatar.
+
+A token is automatically generated for you each time the RoboSats webpage is accessed. Users may re-generate as many randomized tokens as they'd like, which is strongly encouraged after each trade. As a note, you can enter any token created by yourself instead of relying on RoboSats. Because only you know the token, it is wise to safely back up your token.
+
+Under the hood, token creation in the RoboSats homepage is the process of generating and encrypting your PGP private key with your token on the client side of the app. The end-user requests from the RoboSats node an avatar and nickname generated from your encrypted token and returns to you the corresponding robot identity. See the graphic below:
+
+![RoboSats Identity Generation Pipeine](https://learn.robosats.com/assets/images/private/usergen-pipeline.png)
```
Comment (okada-blue) [inline docs/_pages/docs/03-understand/02-robots.md:51]: Misspelled "Pipeline" in the brackets. Missing an "L" and should read:

`RoboSats Identity Generation Pipeline`

Comment (Reckless-Satoshi) [reply docs/_pages/docs/03-understand/02-robots.md:51]: Do you want to push a new commit to your branch fixing this typo and remove the "Include WIP" line? Then I will merge :)

**Comment by Reckless-Satoshi** [comment]
> > Please submit an invoice for 20K Sats
> 
> Woops, I provided the invoice on the commit comment. I've copied here as well: lnbc200u1p3s5v32pp5p62jxach58d48u6yg4tt7jms9pvs7m07j4m7dzk2stud6pa62u8qdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5r7ta253wcwp8usfjdqf5tja474wx877lu49puvqhn5gktud50hvs9qyyssq3cygts9l8r2rhzrv89a6d4wz8ect8296fsp60y32y4k00ghav5fkqm8dvlgme49mjvvcn0a3s8kuw7mzxs4jkltjtn0g4ufxm28c5xqp2txpxu

23e5c1d22cc0521cee93fb0fed949dfd195195f55623a96558d63e19972009cb

File: docs/_pages/docs/03-understand/02-robots.md (hunk 2 above)
Comment (okada-blue) [reply docs/_pages/docs/03-understand/02-robots.md]: I _think_ I pushed a new commit correctly.. let me know if I've not though. Sorry, full noob on display

**Pull request merged**